igcollect/strongswan.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""igcollect - Strongswan stats
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
```

Comment:
You are shadowing the module `time` with that import. For me it is fine, but in the future one needs to think about it.

Reply:
As discussed on Slack, I consider shadowing good for this simple script.